c31-bywheat/by-wheat-nutshell-react
# Cm messages

# Description

Implemented input field that after submit button is clicked. Takes the value of the field save it to the database and refreshes the state.

Applies to issue # 5

## Type of change

Please delete options that are not relevant.




# Testing Instructions

Run json and npm start
Click on messages tab of the nav bar.
You should be able to type and new message send it and have it appear on the dom.
You should also be able to delete any message



# Checklist:
